### ISSUE TITLE
IDEA - UserPage: Move inline JavaScript to external files

### ISSUE THREAD

**Issue by wolfgangs-code** [Intro-to-SE-lab-Spring-22/Group-8#67]
This would clean up maintaining and navigating the page a **lot!**

**Comment by adb1042** [comment]
of course! the reason its where it is right now was for my sake on the programming side of things, trying to keep things simple until i could get it functional and integrated with your back-end stuff. once the integration was complete i fully planned to move it
